docs/docsite/rst/vmware/scenario_remove_vm.rst

```diff
@@ -0,0 +1,127 @@
+.. _vmware_guest_remove_virtual_machine:
+
+*****************************************
+Remove an existing VMware virtual machine
+*****************************************
+
+.. contents:: Topics
+
+Introduction
+============
+
+This guide will show you how to utilize Ansible to remove an existing VMware virtual machine.
+
+Scenario Requirements
+=====================
+
+* Software
+
+    * Ansible 2.5 or later must be installed.
+
+    * The Python module ``Pyvmomi`` must be installed on the Ansible (or Target host if not executing against localhost).
+
+    * Installing the latest ``Pyvmomi`` via pip is recommended [as the OS packages are usually out of date and incompatible].
```

Comment:
I like to provide the command whenever possible. I'd change this from "Installing the latest `pyvmomi` via pip is recommended" to "We recommend installing the latest version with pip: `pip install Pyvmomi`".

Also, the comment about OS packages should probably be in (these) instead of [these].

Reply:
Done.